pxtlib/sprite-editor/spriteEditor.ts

```diff
@@ -386,6 +395,35 @@ namespace pxtsprite {
                 this.altDown = true;
             }
 
+            if (this.activeTool === PaintTool.Marquee && this.state.floatingLayer) {
+                let didSomething = true;
+                switch (event.keyCode) {
+                    case 8: // Backspace
```

Comment:
backspace is closing the image editor for me and deleting the block, needs ``event.stopPropgation`` i think